app/grants/models/grant.py

```diff
@@ -60,6 +60,14 @@ class GrantPayout(SuperModel):
         default='DAI',
         help_text=_('Currency in which funds would be paid')
     )
+    conversion_rate = models.PositiveIntegerField(
+        null=True,
+        blank=True
+    )
+    conversion_date = models.DateTimeField(
+        null=True,
+        blank=True
+    )
```

Comment:
Don't we need a fk to Token table here as well ?

Reply:
yea i think so too